packages/flutter/test/material/about_test.dart

```diff
@@ -745,10 +743,10 @@ class AboutDialogObserver extends NavigatorObserver {
   int dialogCount = 0;
 
   @override
-  void didPush(Route<dynamic> route, Route<dynamic> previousRoute) {
+  void didPush(Route<dynamic> route, Route<dynamic>? previousRoute) {
     if (route.toString().contains('_DialogRoute')) {
       dialogCount++;
     }
     super.didPush(route, previousRoute);
   }
-}
+}
```

Comment:
nit: your editor seems to remove these from all files? I think our style is to leave these in general?

Reply:
No, it was a script I wrote.  I'll add them back in.